nix/tools/withTools.nix

```diff
@@ -348,20 +348,22 @@ let
       ''
         export PGRST_SERVER_UNIX_SOCKET="$tmpdir"/postgrest.socket
 
-        rm -f result
-        if [ -z "''${PGRST_BUILD_CABAL:-}" ]; then
-          echo -n "Building postgrest (nix)... "
-          # Using lib.getBin to also make this work with older checkouts, where .bin was not a thing, yet.
-          nix-build -E 'with import ./. {}; pkgs.lib.getBin postgrestPackage' > "$tmpdir"/build.log 2>&1 || {
-            echo "failed, output:"
-            cat "$tmpdir"/build.log
-            exit 1
-          }
-          PGRST_CMD=$(echo ./result*/bin/postgrest)
-        else
-          echo -n "Building postgrest (cabal)... "
-          postgrest-build
-          PGRST_CMD=postgrest-run
+        if [ -z "''${PGRST_CMD:-}" ]; then
```

Comment:
How about using argbash `ARG_USE_ENV` instead?

There's an example on https://github.com/PostgREST/postgrest/blob/cc4f92005c401358d2f84e40d45c7a8fc4633faa/nix/tools/withTools.nix#L28-L32

Reply:
And have it documented too! Thanks for the suggestion, will do!